pyspecdata/general_functions.py

```diff
@@ -128,13 +128,38 @@ def check_ascending_axis(u,tolerance = 1e-7,additional_message = []):
     assert du > 0, thismsg
     return du
 
-def init_logging(level=logging.INFO, filename='pyspecdata.log'):
+def init_logging(levelString, filename='pyspecdata.log'):
```

Comment:
First, AWESOME KUDOS for starting a pull request on pyspecdata!!

Reply:
yes! much more organized!